spec/unit_tests/moab/storage_object_validator_spec.rb

```diff
@@ -84,4 +84,23 @@
       expect(verification_array.empty?).to eq true
     end
   end
+
+  context '#identify_druid' do
+    let(:same_druid) { 'bz514sm9647' }
+    let(:same_druid_path) { 'spec/fixtures/storage_root01/moab_storage_trunk/bz/514/sm/9647/bz514sm9647' }
+    let(:storage_obj1) { Moab::StorageObject.new(same_druid, same_druid_path) }
+    let(:storage_obj_validator1) { described_class.new(storage_obj1) }
+    let(:different_druid) { 'jj925bx9565' }
+    let(:different_druid_path) { 'spec/fixtures/storage_root01/moab_storage_trunk/jj/925/bx/9565/jj925bx9565' }
+    let(:storage_obj2) { Moab::StorageObject.new(different_druid, different_druid_path) }
+    let(:storage_obj_validator2) { described_class.new(storage_obj2) }
+
```

Comment:
i think for this it might be easier to read if you declare the variables inside the `it ..... do.......end ` block rather than declaring all the variables at the top... but if you don't agree then i'm ok with how it is